marathon/client.py

```diff
@@ -373,7 +373,13 @@ def batch(iterable, size):
             params = {'scale': scale}
             if host: params['host'] = host
             response = self._do_request('DELETE', '/v2/apps/{app_id}/tasks'.format(app_id=app_id), params)
-            return self._parse_response(response, MarathonTask, is_list=True, resource_name='tasks')
+            # Marathon is inconsistent about what type of object it returns on the multi
+            # task deletion endpoint, depending on the version of Marathon. See:
+            # https://github.com/mesosphere/marathon/blob/06a6f763a75fb6d652b4f1660685ae234bd15387/src/main/scala/mesosphere/marathon/api/v2/AppTasksResource.scala#L88-L95
+            if response.json().has_key("tasks"):
+                return self._parse_response(response, MarathonTask, is_list=True, resource_name='tasks')
+            else:
+                return response.json()
```

Comment:
I can't. The response back is not a deployment, it is a kind of scaling stub that we don't have a model for.